### DISCUSSION TITLE
The monitoring doesn't show any job

### DISCUSSION THREAD

**Discussion by spypix3l** [romanzipp/Laravel-Queue-Monitor#96] [Q&A]
Hello,

I hope I'm posting at the right place. I'm trying to put your package in place to monitor the my job queue, but without success.
I went throught the whole process explained [here ](https://github.com/romanzipp/Laravel-Queue-Monitor)
Everything seem to have run smoothly (except a few depreciation warnings in return of the composer command line).
I've also added the route to my web route file, allowing me to see the UI at https://www.mywesite.com/jobs

However, it shows no job at all on the page, when I have many in my "jobs" and "failed_jobs" tables.
I wish I could share more information with you, but since it doesn't show anything, no error message, etc, I don't really know where to look at, and what could be disfunctioning. Do you have any idea of what could be the problem, or any advice toto help me identify the problem?

Thank you very much,

**Comment by geanruca** [comment]
It happens the same to me after installing Laravel Horizon
